test/gtest/graphapi_conv_bias_res_add_activ_fwd.cpp

```diff
@@ -45,13 +46,7 @@
 namespace gr = miopen::graphapi;
 namespace conv_graph_api_test {
 
-bool IsTestSupportedForDevice()
-{
-    using e_mask = enabled<Gpu::gfx94X, Gpu::gfx103X, Gpu::gfx110X>;
-    // gfx120X is not enabled due to WORKAROUND_SWDEV_479810
-    using d_mask = disabled<Gpu::None>;
-    return ::IsTestSupportedForDevMask<d_mask, e_mask>();
-}
+bool IsTestSupportedForDevice() { return ::IsDeviceSupportedForCK(); }
```

Comment:
How do we know that `graphapi` use only `CK`? There should be no dependency here.

Reply:
The graphAPI implementation in this case is using the fused CK solver.
I agree that it's not really clear, but not really sure how else we can avoid this issue.

Reply:
Oh you mean you would prefer to have just the checks inside the test, and not have the shared file with fused?

I combined it since they both have the same requirements, but could split it back out if you think that's better.

Reply:
Yes, inside _this_ test. Also let's keep that file for CK.

Reply:
Do these changes match what you were expecting?